target/linux/ramips/image/mt7621.mk

```diff
@@ -228,6 +228,18 @@ define Device/iodata_wn-gx300gr
 endef
 TARGET_DEVICES += iodata_wn-gx300gr
 
+define Device/iodata_wnpr2600g
+  DTS := WNPR2600G
```

Comment:
Already suggested this in another PR:
I wonder whether it wouldn't be desirable to start with improving the naming of DTSes, e.g.
iodata_wnpr2600g.dts
mt7621_iodata_wnpr2600g.dts

However, this is just a suggestion without any obligation (don't even know whether devs will like it).

Reply:
I think we could probably change the default image template like ath79 before applying this naming scheme.

Reply:
I also like the style in ath79 target.
It is useful because it is easy to see which SoC the device has and is grouped by it in the list.

However, (if we do,) it may be better to do this collectively within another patchset rather than changing it with one model.

Reply:
@981213 Yes, that would be obviously even better. Who will do it?
@musashino205 Okay, no objection if you keep the current state ...

Reply:
@adrianschmutzler I could do that the week after next if you don't beat me to it :)

Reply:
Well, I just started and will wait for some feedback from devs now:
https://github.com/openwrt/openwrt/pull/2171